src/main/scala/io/phdata/retirementage/loadgen/LoadGeneratorConfig.scala

```diff
@@ -28,5 +28,8 @@ class LoadGeneratorConfig(args: Array[String]) extends ScallopConf(args) {
   val dimName  = opt[String]("dim-name", required = false, default = Some("dimloadtest"))
   val subName  = opt[String]("subdim-name", required = false, default = Some("subloadtest"))
 
+  val outputFormat = opt[String]("output-format", required = true, default = Some("parquet"))
+  val kuduMasters  = opt[String]("kudu-masters", required = false, default = Some("localhost:7051"))
```

Comment:
Remove the default from this. It's better they forget to set it and it blows up than tries to connect to localhost.